src/test/java/server/IntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+package server;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class IntegrationTest {
```

Comment:
**praise** I like the inclusion of an integration test!

**question**: This is a pretty generic name. Is there a more specific name for this? Is there a concept implied here that we haven't made explicit?

Reply:
I wanted to call it HTTPServerTest but there's already a unit test with the same name. Is there a better way to differentiate the two (`HTTPServerIntegrationTest` ?), or are you seeing a concept I'm missing?

Reply:
`HTTPServerIntegrationTest` feels like that would be specific to any application that you configure your HTTP server for.

This feels more like an integration test for the app designed for the `http_server_spec`.